.github/workflows/scorecard.yml

```diff
@@ -46,6 +46,6 @@ jobs:
           retention-days: 5
       
       - name: "Upload to code-scanning"
-        uses: github/codeql-action/upload-sarif@4c8f13758e748234abaa6a831d6f53981844a524 # tag=v2.1.26
```

Comment:
This looks malicious that commit is updated but the tag is not.

Reply:
Updated.

Reply:
There are other version comments that are incorrect. I will create a PR to fix them after merge this one.